node/MIGRATION.md

```diff
@@ -10,9 +10,22 @@ chan
     .send(options, a1, a2, a3, cb);
 ```
 
-After:
+After (simple):
 ```
 chan
-    .request(options, cb)
+    .request(options)
+    .send(a1, a2, a3, cb);
+```
+
+After (if needed/useful):
+```
+chan
+    .request(options)
+    .on('error', function onError(err) {
+        console.error('too bad:', err);
+    })
+    .on('response', function onResponse(res) {
+        console.log('got:', res.arg2, res.arg3);
+    })
```

Comment:
And the even long story that we’re shooting for is streaming request and streaming response, so instead of request.send(a1, a2, a3), we’ll have request.arg1.end(name); request.arg2.end(headers); body.pipe(request.arg3), and similar for response.arg* streams. Is this correct?